doc/modules/tree.rst

```diff
@@ -534,6 +534,45 @@ Mean Absolute Error:
 
 where :math:`X_m` is the training data in node :math:`m`
 
+.. _minimal_cost_complexity_pruning
+
+Minimal Cost-Complexity Pruning
+===============================
+
+Minimal cost-complexity pruning is an algorithm used to prune a tree after it
+is grown. This algorithm is parameterized by :math:`\alpha\ge0` known as the 
+complexity parameter. The complexity parameter is used to define the 
+cost-complexity measure, :math:`R_\alpha(T)` of a given tree :math:`T`:
+
+.. math::
+
+  R_\alpha(T) = R(T) + \alpha|\tilde{T}|
+
+where :math:`|\tilde{T}|` is the number of terminal nodes in :math:`T` and 
+:math:`R(T)` is traditionally defined as the total misclassification rate of 
+the terminal nodes. Alternatively, scikit-learn uses the total sample weighted 
+impurity of the terminal nodes for :math:`R(T)`. As shown in the previous 
+section, the impurity of a node depends on the criterion. Minimal 
+cost-complexity pruning finds the subtree of :math:`T` that minimizes 
+:math:`R_\alpha(T)`.
+
+The cost complexity measure of a single node is 
+:math:`R_\alpha(t)=R(t)+\alpha`. The branch, :math:`T_t`, is defined to be a 
+tree where node :math:`t` is its root. In general, the impurity of a node
+is greater than the sum of impurities of its terminal nodes, 
+:math:`R(T_t)<R(t)`. However, the cost complexity measure of a node, 
+:math:`t`, and its branch, :math:`T_t`, can be equal depending on 
+:math:`\alpha`. We define the effective :math:`\alpha` of a node to be the 
+value where they are equal, :math:`R_\alpha(T_t)=R_\alpha(t)` or 
+:math:`\alpha_{eff}=(R(t)-R(T_t))/(|\tilde{T}|-1)`. A non-terminal node with 
```

Comment:
nit: since `\alpha_{eff}` also depends on `t`, shouldn't it be more like `\alpha_{t}^{eff}` or something?

Reply:
and this?